### PR TITLE
contributing doc

### PR DESCRIPTION
**Pull request by tarasglek**
Feel free to add cloudflare details to this

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+- [Documentation](#documentation)
+- [Issue and Pull Request Guidelines](#issue-and-pull-request-guidelines)
+- [Community](#community)
+- [License](#license)
+
+## Getting Started
+
+To get started with contributing to ChatCraft.org, please make sure you have the following:
+
+- [pnpm](https://pnpm.io) package manager installed globally
```
Comment (humphd) [inline CONTRIBUTING.md:22]: Did you want to switch to pnpm from yarn?  That's fine with me.  We'll have to update our scripts, lock file, etc.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+## Code Style
+
+We follow a specific code style to maintain consistency throughout the project. Please make sure your code adheres to the following guidelines:
+
+- Use consistent indentation (preferably 2 spaces).
+- Follow the naming conventions used in the existing codebase.
+- Write clear and concise comments when necessary.
+- Keep lines of code within 80 characters.
+
```
Comment (humphd) [inline CONTRIBUTING.md:62]: Add a note to use `prettier` via VSCode or run the `prettier` script to auto-format.  We also format on commit with Husky.

**Review by humphd — CHANGES_REQUESTED** [review]
Thanks for starting this

File: CONTRIBUTING.md (hunk 1 above)
Comment (tarasglek) [reply CONTRIBUTING.md:22]: i been using pnpm for most of chatcraft history

Comment (humphd) [reply CONTRIBUTING.md:22]: Landed https://github.com/tarasglek/chatcraft.org/pull/193, so now we're using pnpm!

**Comment by tarasglek** [comment]
```


➤ YN0070: Migrating from Yarn 1; automatically enabling the compatibility node-modules linker 👍
--
14:52:01.351 |  
14:52:01.499 | ➤ YN0000: ┌ Resolution step
14:52:04.195 | ➤ YN0061: │ rollup-plugin-terser@npm:7.0.2 is deprecated: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-terser
14:52:05.658 | ➤ YN0061: │ sourcemap-codec@npm:1.4.8 is deprecated: Please use @jridgewell/sourcemap-codec instead
14:52:08.151 | ➤ YN0032: │ fsevents@npm:2.3.2: Implicit dependencies on node-gyp are discouraged
14:52:12.850 | ➤ YN0061: │ rollup-plugin-inject@npm:3.0.2 is deprecated: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-inject.
14:52:12.888 | ➤ YN0032: │ better-sqlite3@npm:8.4.0: Implicit dependencies on node-gyp are discouraged
14:52:19.129 | ➤ YN0002: │ chatcraft.org@workspace:. doesn't provide @codemirror/view (p72df8), requested by @uiw/react-codemirror
14:52:19.130 | ➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
14:52:19.136 | ➤ YN0000: └ Completed in 17s 636ms
14:52:19.225 | ➤ YN0000: ┌ Post-resolution validation
14:52:19.226 | ➤ YN0028: │ The lockfile would have been modified by this install, which is explicitly forbidden.
14:52:19.226 | ➤ YN0000: └ Completed
14:52:19.226 | ➤ YN0000: Failed with errors in 17s 728ms


```

not sure why this happens

**Pull request merged**